Classes/GlobalStateExplorers/DatabaseBrowser/FLEXSQLiteDatabaseManager.m

```diff
@@ -164,7 +169,7 @@ - (FLEXSQLResult *)executeStatement:(NSString *)sql arguments:(NSDictionary *)ar
         }
         
         if (status == SQLITE_DONE) {
-            if (rows.count) {
+            if (rows.count || columnCount > 0) {
```

Comment:
`columnCount` is usually going to be non-zero here, this is going to cause the logic in the `else` to not execute
```suggestion
            if (rows.count) {
```

Reply:
Is that true? My understanding was for `SELECT` statements it will have a positive count, for update/delete/insert it returns 0?
If that's not true then I think you are right.

Reply:
Whoops, you are correct! I'm gonna add that as a comment for my future self